### DISCUSSION TITLE
[macOS] where is the rio log file stored?

### DISCUSSION THREAD

**Discussion by andrewcrook** [raphamorim/rio#626] [Q&A] [answered]
I have tried to setup logging 

```
 [developer]
#log-level= “ERROR"
 log-level = “DEBUG"
```

However, I have looked in the common folders and console app for the log file and I cannot find its location.
Any ideas?

**Comment by orhun** [comment]
Not sure if Rio currently saves logs to a file but it would be a really nice feature addition I think.

**Reply by andrewcrook** [reply]
Does it not?
Is there another way to view this logging data that I am missing?
Found a bug just trying to get the data around it.

**Reply by raphamorim** [reply]
Rio doesn't save the log yet 😢 

The log-level property only logs if the application is launched in same process instance (as `cargo run`).

This is indeed a good thing to add for the next version.

**Reply by orhun** [reply]
I will give this a shot very soon.

**Reply by raphamorim** [reply, marked as answer]
Supports to logs have been implemented by @orhun in https://github.com/raphamorim/rio/issues/645 and it’s coming to next version
